questions/javascript-questions.md

```diff
@@ -64,3 +64,4 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What are the benefits of using `spread syntax` and how is it different from `rest syntax`?
 * How can you share code between files?
 * Why you might want to create static class members?
+* At what time will the Settimeout() function be executed, if called at 4000 milliseconds?
```

Comment:
maybe it worth to use the correct name of function? `setTimeOut` ? and (I'm not sure but) looks like in English `at 4000 milliseconds` sounds strange